style.css

```diff
@@ -0,0 +1,48 @@
+* {
+    margin: 0;
+    padding: 0;
+    box-sizing: border-box;
+}
+
+.app {    
+    background-color: #FFFFFF;
+    display: flex;
+    flex-direction: column;
+    align-items: flex-start;       
+    max-width: 1440px;
+    max-height: 704px;     
```

Comment:
You can apply width for a container. But applying height or max-height is not appropriate.